.github/workflows/AtlasReleaseThemeFiles.yml

```diff
@@ -5,63 +5,56 @@ name: Create Release For Atlas Theme Folder
 # upgraded to Atlas 3, a migration doc included a link to a .zip file with theme files.
 
```

Comment:
Same here, @jordanch , @pratikgoyal9 , can we add high level comment what this job does? Why we have to release Atlas Theme Files?